frontend/hub/collections/hooks/useCollectionFilters.tsx

```diff
@@ -26,21 +26,21 @@ export function useCollectionFilters() {
       {
         key: 'keywords',
         label: t('Name'),
-        type: ToolbarFilterType.Text,
+        type: ToolbarFilterType.MultiText,
```

Comment:
MultiText does not work in this filter, should we check (after merge of this PR) and change some of those into SingleText? Im not sure if backend will support this, but you are probably better informed.